src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const employeeList = tbody.querySelectorAll('tr');
+
+function sortList(e) {
+  const column = e.target.cellIndex;
+
+  function getData(employee) {
+    const data = employee.cells[column].innerHTML;
+
+    if (data[0] === '$') {
+      return +data.split(',').join('').slice(1);
+    }
+
+    return data;
+  }
```

Comment:
No need to create function inside another one
```suggestion

function getData(employee) {
    const data = employee.cells[column].innerHTML;

    if (data[0] === '$') {
      return +data.split(',').join('').slice(1);
    }

    return data;
  }

function sortList(e) {
  const column = e.target.cellIndex;

```

Reply:
@IvanFesenko yes, but in this case, we will have function getData with two parameters (getData(employee, column))